go/vt/vtctl/vtctl.go

```diff
@@ -317,6 +317,9 @@ var commands = []commandGroup{
 			{"VerticalSplitClone", commandVerticalSplitClone,
 				"<from_keyspace> <to_keyspace> <tables>",
 				"Start the VerticalSplitClone process to perform vertical resharding. Example: SplitClone from_ks to_ks 'a,/b.*/'"},
+			{"VDiff", commandVDiff,
+				"-workflow=<workflow> <target keyspace> [-source_cell=<cell>] [-target_cell=<cell>] [-tablet_types=REPLICA] [-filtered_replication_wait_time=30s]",
```

Comment:
Does order matter? Does `<target keyspace>` need to be at the end?

Reply:
Order doesn't matter. The ordering of the help line implies that `workflow` is a mandatory flag and `target_keyspace` is a mandatory argument. The rest are optional.